docs/1.4/access-control.md

```diff
@@ -0,0 +1,59 @@
+# FilterPolicy Plugin for Access Control
```

Comment:
This (and a bunch of other files) aren't in the right place. We need to be very careful about this. @steph did you double check all the file locations?